pkg/entrypoint/entrypointer.go

```diff
@@ -77,12 +90,20 @@ func (e Entrypointer) Go() error {
 	if e.Entrypoint != "" {
 		e.Args = append([]string{e.Entrypoint}, e.Args...)
 	}
+	t := time.Now()
```

Comment:
nit: If `t` isn't used anywhere else, you can just inline it on line 96: `time.Now().Format(time.RFC3339)`

Reply:
done.